rvic/convert.py

```diff
@@ -70,17 +72,16 @@ def uhs2param_init(config_file):
                       options['VERBOSE'])
 
     for direc in directories:
-        log.info('%s directory is %s' % (direc, directories[direc]))
+        log.info('%s directory is %s', direc, directories[direc])
```

Comment:
no.  the `logging` module and `log` objects will expand the strings automatically.